www/artists/emmalake/style.css

```diff
@@ -80,6 +80,10 @@ a:hover {
 	text-decoration: none;
 	font-weight: bold;
 }
+.b{
+	color: #242424;
+	text-decoration: none;
+}
```

Comment:
Just to confirm: rather than using the navbar style for the links in the Artist list, you've given them a new class which makes them a dark grey with no underline, correct? I want to ensure this was intentional.